CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 0.17.0 (2021-09-09)
+* Update to contracts 0.40.0rc (https://github.com/raiden-network/raiden-services/pull/1062)
```

Comment:
We also upgraded Raiden. Shouldn't this be mentioned, or does the Raiden upgrade only change the referenced contracts?

Reply:
✔️